lib/client.js

```diff
@@ -20,7 +20,7 @@ class Client {
   /**
    * Get the httpClient's baseURL value.
    *
-   * @return {String}
+   * @return {String} - Get the httpClient baseURL
```

Comment:
since httpClient is internal implementation / private, I think the documentation should mention it.

Reply:
Should we remove all of the httpClient and ReferenceResolver documentation?

Reply:
we did :)

Reply:
fixed.